Library/Formula/pstoedit.rb

```diff
@@ -15,6 +15,21 @@ class Pstoedit < Formula
   depends_on "imagemagick"
   depends_on "xz" if MacOS.version < :mavericks
 
+  # Fix for pstoedit search for plugins, thereby restoring formats that
+  # worked in 3.62 but now don't in 3.70, including PIC, DXF, FIG, and
+  # many others.
+  #
+  # Taken from:
+  # https://build.opensuse.org/package/view_file/openSUSE:Factory/pstoedit/pstoedit-pkglibdir.patch?expand=1
+  #
+  # This patch changes the behavior of "make install" so that:
+  # * If common/plugindir is defined, it checks only that directory.
+  # * It swaps the check order: First checks whether PSTOEDITLIBDIR exists. If
+  #   it exists, it skips blind attempts to find plugins.
+  # As PSTOEDITLIBDIR is always defined by makefile, the blind fallback will
+  # be attempted only in obscure environments.
+  patch :DATA
```

Comment:
Can you please submit this patch to the upstream developers of this project and add a link to the upstream patch submission and explanation of why the patch is needed in a comment in the formula file. Thanks!